### ISSUE TITLE
[Accessibility] MessageBar with Link - color contrast ratio should be at least 4.5

### ISSUE THREAD

**Issue by rsemenov** [OfficeDev/office-ui-fabric-react#1009]
#### Describe the issue:

When MessageBar with messageBarType=MessageBarType.info contains Link the color contrast ratio between background color #f4f4f4 and link color #0078d7 is 4.1.

#### Actual behavior:

Color contrast ratio is 4.1

#### Expected behavior:

Color contrast ratio should be at least 4.5.
https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

### If applicable, please provide a codepen repro:

https://codepen.io/rsemenov/pen/egGQYW

**Comment by rsemenov** [comment]
@battletoilet, I would like to make sure it is not expected behavior so I can submit a pull request.

**Comment by betrue-final-final** [comment]
Fixed with PR #1470
